egs2/l3das22/enh1/local/metric.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+[ -f ./path.sh ] && . ./path.sh
+. parse_options.sh || exit 1;
+
+pip show -f jiwer >/dev/null || pip install jiwer
+pip show -f transformers >/dev/null || pip install transformers
```

Comment:
We may add a note that these evaluation metrics are from the official requirements?

Reply:
Added.